pkg/apis/flowcontrol/validation/validation.go

```diff
@@ -87,7 +87,10 @@ func ValidateFlowSchemaUpdate(old, fs *flowcontrol.FlowSchema) field.ErrorList {
 func ValidateFlowSchemaSpec(spec *flowcontrol.FlowSchemaSpec, fldPath *field.Path) field.ErrorList {
 	var allErrs field.ErrorList
 	if spec.MatchingPrecedence <= 0 {
```

Comment:
This conflicts with the decision to make the MatchingPrecedence a non-pointer -- the test should be "< 0" now. But changing the type to uint32 would give you that check for free, I think...

Reply:
To be clear, we are currently not making it a pointer.

Reply:
@jktomer with the defaulting in place, there is no real difference between `< 0` and <= 0`.